packages/components/navbar/src/NavbarItem/NavbarItem.tsx

```diff
@@ -83,6 +84,16 @@ function _NavbarItem(
     </Comp>
   );
 
+  if (!title) {
+    item = (
+      <div>
+        <Tooltip content={label} placement="bottom" showDelay={600} usePortal>
+          {item}
```

Comment:
Is this correct? We are re-assigning the `item` value on line 88 and then use it again from within here 🤔

Reply:
Yes that's possible, like `i = i + 1`.

Reply:
I see 👍